src/panels/KeybindsPanel/ListLayout.tsx

```diff
@@ -8,10 +8,15 @@ import { Layout } from '@/components/Layout/Layout';
 import { useAppSelector } from '@/redux/hooks';
 import { Action, KeybindInfoType, KeybindModifiers } from '@/types/types';
 
-import { KeybindButtons } from './KeybindButtons';
 import { KeybindInfo } from './KeybindInfo';
+import { ListEntry } from './ListEntry';
 
-export function ListLayout() {
+interface Props {
+  // The height of this view, used to get the scrolling behavior right
+  height: number;
+}
+
+export function ListLayout({ height }: Props) {
```

Comment:
This component doesn't need to know the height. It is sufficient if the parent div has the height and the outermost div in this component is "100%".